### DISCUSSION TITLE
Add labels to logs view in "Live" view mode

### DISCUSSION THREAD

**Discussion by avkonst** [grafana/grafana#56849] [Observability]
**What would you like to be added**:

I would like to see labels per log line when loki is queried in "Live" mode.

**Why is this needed**:

When I watch for logs from multiple containers in Live mode, all logs go to the single list and it is impossible to know a container for each log line. Current workaround is to open multiple grafana instances with unique filter for all containers of interest.

**Comment by dprokop** [comment]
@aocenas is there currently any other way to improve this workflow rather than having multiple tabs with unique filters?

**Comment by aocenas** [comment]
well showing labels in the live tail could make sense I don't think there is any technical reason not to do that.

**Comment by aocenas** [comment]
Need to be checked if loki sends labels with the live tailing API, it's possible it doesn't and the only thing we know is that they mach the query labels but don't now any other unique ones.

**Comment by ivanahuckova** [comment]
I have looked into response and we actually get all labels, so there is no technical reason on why not to do that.

**Comment by JuniorJPDJ** [comment]
This would be very nice addition.
When having more than just a few containers making one bigger service, it's getting really hard to follow logs. If you also have replicas of the same container, logs are getting even more hard to distinguish.

**Comment by easy2stake** [comment]
Hi,

Is there any solution or workaround to this ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
